web/app/themes/mitlib-parent/inc/footer-main.php

```diff
@@ -1,23 +1,20 @@
 <?php
 /**
- * The template for the main footer, showing EDS / Barton as discovery options.
+ * The template for the main footer, showing Alma / Primo as discovery options.
  *
- * @package MITlib_Parent
- * @since 0.0.1
+ * @package MIT_Libraries_Parent
+ * @since 1.11.0
  */
 
-namespace Mitlib\Parent;
```

Comment:
Ditto for this - while I don't think anything in this template calls anything namespaced at the moment, I'd like to keep this line in place for consistency and in case we ever do need to call any PHP functions down the road.

Reply:
@matt-bernhardt I un-removed the namespace for this file as you requested but also added it to the other file where it wasn't before. I'm fairly certain that's probably fine but also if not let me know!